### PR TITLE
Add feature flag support_and_triggers

### PR DESCRIPTION
**Pull request by de-lan**
Adding the feature flag: support_and_triggers, targeting 6.8.0

### CODE REVIEW

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -5085,6 +5085,14 @@
     </description>
   </property>
 
+  <property>
+    <name>feature.support.and.triggers.enabled</name>
```
Comment (seanfeld) [inline cdap-common/src/main/resources/cdap-default.xml]: can you add more context to this feature flag name? maybe feature.pipelines.and.triggers.enabled ? I'll send you the naming guideline doc on chat

File: cdap-common/src/main/resources/cdap-default.xml [hunk 2]
```diff
@@ -5085,6 +5085,14 @@
...
+    <value>false</value>
+    <description>
+      Enables andding AND or OR trigger groups for a pipeline.
```
Comment (seanfeld) [inline cdap-common/src/main/resources/cdap-default.xml]: adding*

**Review by seanfeld — CHANGES_REQUESTED** [review]
also add @sechegaray for feature flag approval

File: cdap-common/src/main/resources/cdap-default.xml (hunk 1 above)
Comment (de-lan) [reply cdap-common/src/main/resources/cdap-default.xml]: Thanks! updated

File: cdap-common/src/main/resources/cdap-default.xml [hunk 3]
```diff
@@ -5085,6 +5085,14 @@
     </description>
   </property>
 
+  <property>
+    <name>feature.pipeline.and.triggers.enabled</name>
```
Comment (seanfeld) [inline cdap-common/src/main/resources/cdap-default.xml]: Sorry this is still confusing - can it be `feature.pipeline.triggers.conditional.and.enabled`? that way someone won't be able to confuse the `and` as part of the description and triggers is towards the front providing more context.

Comment (de-lan) [reply cdap-common/src/main/resources/cdap-default.xml]: Yeah I have to say this `and` is easy to confuse with. I'll update

**Review by seanfeld — APPROVED** [review]

**Review by seanzhougoogle — APPROVED** [review]

File: cdap-features/src/main/java/io/cdap/cdap/features/Feature.java [hunk 4]
```diff
@@ -27,6 +27,7 @@
  */
 public enum Feature {
   REPLICATION_TRANSFORMATIONS("6.6.0");
```
Comment (sechegaray) [inline cdap-features/src/main/java/io/cdap/cdap/features/Feature.java]: Please change this semicolon to a comma

Comment (de-lan) [reply cdap-features/src/main/java/io/cdap/cdap/features/Feature.java]: Thanks! Updated

**Pull request merged**